recOrder/plugin/main_widget.py

```diff
@@ -1160,7 +1159,6 @@ def handle_bire_image_update(self, value: NDArray):
             retardance=value[0],
             orientation=value[1],
             ret_max=np.percentile(value[0], 99.99),
-            mode=self.acq_mode,
```

Comment:
How about the `show_info` statement below? If the new 3D overlay implementation is still significantly slower than 2D I guess we still want it around?

Reply:
Thanks for mentioning it. During my testing today the overlay was noticeably faster than before but still had a noticeable delay, so I still like having the `show_info` statement.